src/CMakeLists.txt

```diff
@@ -381,6 +381,10 @@ ENDIF(UNIX)
 # will be used.
 ADD_DEFINITIONS(-DLUA_DEFAULT_PATH="${LUA_DEFAULT_DIR}")
 
+# Enable the built-in updater by default. Linux packagers will find it useful to disable it
+# since package managers are responsible for updates there
+ADD_DEFINITIONS(-DINCLUDE_UPDATER=true)
```

Comment:
Do you need to include a value - the presence of the `#define` is sufficient to trigger `#if defined(INCLUDE_UPDATER)` code - unless you plan to actually _test_ the value as well with `#if defined(INCLUDE_UPDATER) && (INCLUDE_UPDATER)` {if that _is_ the right way to do it... 😕}, and I get the feeling from a bit of searching that you should use `TRUE` not `true`...